File: Python/global_enum.py
from enum import Enum

class MessageTypeEnum(Enum):
    TERMINATION = 0
    PING = 1
    DATA = 2
    COMMAND = 3
    FEEDBACK = 4
    READY = 5

class InputsEnum(Enum):
    MOVE_LEFT = 0
    MOVE_RIGHT = 1
    MOVE_UP = 2
    MOVE_DOWN = 3

File: GlobalEnum.cs
public static class GlobalEnums
{
    public enum InputsEnum
	{
		MOVE_LEFT,
		MOVE_RIGHT,
		MOVE_UP,
		MOVE_DOWN
	}

	public enum MessageTypeEnum
	{
		TERMINATION,
		PING,
		DATA,
		COMMAND,
		FEEDBACK,
		READY
	}

	public enum AgentStateEnum
	{
		ACT,
		OBSERVE,
		WAIT
	}
}